test-crates/.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[env]
+CLASSPATH = { value = "duchess-java-tests/target/java", relative = true, force = true }
```

Comment:
Instead of adding this file, could you add a `println!("cargo:rustc-env=CLASSPATH=...");` line to the `build.rs` of `duchess-java-tests`?

Each crate in this `test-crates` currently requires its own `CLASSPATH` value, but this `.cargo/config.toml` is currently declaring a global one. The tests of `viper` are not failing just because the `CLASSPATH` declared in `viper/build.rs` seems to have precedence, but I'd prefer to not rely on that.